src/main.cpp

```diff
@@ -292,7 +292,7 @@ struct CNodeState {
     //! Were the first headers requested in a sync received
     bool fFirstHeadersReceived;
     //! Since when we're stalling block download progress (in microseconds), or 0.
-    int64_t nStallingSince;
+    //  int64_t nStallingSince; // BU: no longer used
```

Comment:
@gandrewstone as requested, I cannot see how this change can hurt given the assumption that the req. mgr. does the same job.

However I strongly agree with @deadalnix that the commented code should simply be removed.  Lots of commented code is immensely distracting and given everything is available in history I would much prefer to simply see the lines removed outright.